.github/dependabot.yml

```
@@ -36,6 +36,10 @@ updates:
       # @see https://github.com/dequelabs/axe-core/issues/3771
       - dependency-name: 'esbuild'
         versions: ['>=0.11.0']
+      # Prevent colorjs.io issue caused by >v0.4.3
+      # @see https://github.com/dequelabs/axe-core/issues/4428
+      - dependency-name: 'colorjs.io'
+        version: ['>0.4.3']
```

This should resolve the check failure:

```suggestion
        versions: ['>0.4.3']
```